utils/rpms/cart.spec

```diff
@@ -57,6 +58,7 @@ Requires: libfabric-devel
 # can't do this until we can land ompi@PR-10 and
 # scons_local@bmurrell/ompi-env-module
 #Requires: ompi-devel
```

Comment:
Should be able to remove these 3 commented out lines now.

Reply:
yeah, we can